index.bs

```diff
@@ -2789,6 +2789,50 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats ### {#signature-attestation-types}
+    - For COSEAlgorithmIdentifier -7 (ES256), `sig` contains a pair of 32-byte integers,
+        R followed by S, generated with ECDSA using Curve P-256 and SHA256 encoded as
+        ASN.1 DER format defined in [[ANSI.X9-62.2005]] and [[FIPS.186-4.2013]].
+
+        Note: As CTAP1/U2F devices are already producing signatures in this format, CTAP2
+        devices will also produce signatures in same format for consistency. Otherwise,
+        newer attestation formats MUST use COSE_KEY signature formats as defined in
+        [[IANA-COSE-ALGS-REG]].
+
+        ```
+        Example:
+        30 44                               ; SEQUENCE (44 Bytes)
+            02 20                            ; INTEGER (20 Bytes)
+            |  3d 46 28 7b 8c 6e 8c 8c  26 1c 1b 88 f2 73 b0 9a
+            |  32 a6 cf 28 09 fd 6e 30  d5 a7 9f 26 37 00 8f 54
+            02 20                            ; INTEGER (20 Bytes)
+              4e 72 23 6e a3 90 a9 a1  7b cf 5f 7a 09 d6 3a b2
+              17 6c 92 bb 8e 36 c0 41  98 a2 7b 90 9b 6e 8f 13
+        ```
+    - For COSEAlgorithmIdentifier -257 (RS256), `sig` contains the signature generated using the
+        RSASSA-PKCS1-v1_5 signature scheme defined in [[RFC8017]] with SHA256 as the hash function.
+        ```
+        Example:
+        87 38 c6 f7 7d 53 3a 7e  73 27 2e 4b 9d 45 c7 19
```

Comment:
This example is just 2,048 random bits, right? Dunno how helpful that is :)

Reply:
Yeah.. Not helpful. Lets remove that.